library/analytics/src/main/java/org/cru/godtools/analytics/AppSeeAnalyticService.java

```diff
@@ -0,0 +1,111 @@
+package org.cru.godtools.analytics;
+
+import android.app.Activity;
+import android.app.Application;
+import android.content.Context;
+import android.os.Bundle;
+
+import com.appsee.Appsee;
+import com.appsee.AppseeListener;
+import com.appsee.AppseeScreenDetectedInfo;
+import com.appsee.AppseeSessionEndedInfo;
+import com.appsee.AppseeSessionEndingInfo;
+import com.appsee.AppseeSessionStartedInfo;
+import com.appsee.AppseeSessionStartingInfo;
+import com.crashlytics.android.Crashlytics;
+
+import org.cru.godtools.analytics.model.AnalyticsActionEvent;
+import org.cru.godtools.analytics.model.AnalyticsScreenEvent;
+import org.greenrobot.eventbus.EventBus;
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
+
+import java.util.Map;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.UiThread;
+
+public class AppSeeAnalyticService implements AnalyticsService, Application.ActivityLifecycleCallbacks, AppseeListener {
+
+    @Nullable
+    private static AppSeeAnalyticService sInstance;
+
+    private AppSeeAnalyticService(Context context) {
+        EventBus.getDefault().register(this);
+        ((Application) context.getApplicationContext()).registerActivityLifecycleCallbacks(this);
+        Appsee.addAppseeListener(this);
+    }
+
+    @NonNull
+    public static synchronized AppSeeAnalyticService getInstance(Context context) {
+        if (sInstance == null) {
+            sInstance = new AppSeeAnalyticService(context);
+        }
+        return sInstance;
+    }
+
+    @UiThread
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onAnalyticScreenEvent(@NonNull final AnalyticsScreenEvent event) {
+        Appsee.startScreen(event.getScreen());
+    }
+
+    @Subscribe
+    public void onAnalyticActionEvent(@NonNull final AnalyticsActionEvent event) {
+        Map<String, Object> data = (Map<String, Object>) event.getAttributes();
+        if (data != null) {
+            Appsee.addEvent(event.getAction(), data);
```

Comment:
how do these events show up? do we need to add an Appsee analytics type and filter action events to only send when they are for the Appsee analytics system?